src/mranderson/util.clj

```diff
@@ -155,6 +155,7 @@
 
 (defn remove-subdirs [dirs]
   (->> (sort dirs)
+       distinct
```

Comment:
In order for the `distinct` to be as useful as possible, `dirs` should be absolutized (or canonicalized? even better) beforehand

Reply:
That's a good point. I think the dirs do come in as fully qualified, but the current code leaves one wondering if that's the case.